pkg/api/types.go

```diff
@@ -898,7 +898,10 @@ const (
 )
 
 // ServiceStatus represents the current status of a service
-type ServiceStatus struct{}
+type ServiceStatus struct {
+	// If the service is assigned a cluster-wide port, it is set here
+	ServicePort int `json:"servicePort,omitempty"`
```

Comment:
This has to be handled like Portal IP - it must be part of Spec.  Specifically, Status must be "observable" information not desired state.

Suggest going back to the name we had when this sort of existed originally - ProxyPort.